.github/workflows/watch-dependencies.yml

```diff
@@ -46,30 +46,33 @@ jobs:
           echo "::set-output name=tag::$latest_tag"
 
       # ref: https://github.com/jacobtomlinson/gha-read-helm-chart
-      - name: Read appVersion of dask chart
+      - name: Read dask Chart.yaml
         id: dask_chart
         uses: jacobtomlinson/gha-read-helm-chart@0.1.3
         with:
           path: dask
-      - name: Read appVersion of daskhub chart
-        id: daskhub_chart
-        uses: jacobtomlinson/gha-read-helm-chart@0.1.3
-        with:
-          path: daskhub
 
+      # Currently we update:
+      #
+      # - dask/Chart.yaml's appVersion
+      # - dask/values.yaml images' tag
+      # - dask/README.md
+      # - daskhub/Chart.yaml's appVersion
+      #
+      # FIXME: Maybe we shouldn't set the daskhub chart's appVersion to the dask
+      #        image's version as we do here. Maybe one could set it to a
+      #        combination of the jupyterhub and dask-gateway Helm charts'
+      #        versions.
+      #
       # ref: https://github.com/jacobtomlinson/gha-find-replace
-      - name: Replace old appVersion with new in dask chart
-        uses: jacobtomlinson/gha-find-replace@2.0.0
-        with:
-          include: "**/dask/*"
-          find: "${{ steps.dask_chart.outputs.appVersion }}"
-          replace: "${{ steps.latest.outputs.tag }}"
-      - name: Replace old appVersion with new in daskhub chart
+      - name: Replace old ghcr.io/dask/dask tag with new
         uses: jacobtomlinson/gha-find-replace@2.0.0
         with:
-          include: "**/daskhub/*"
-          find: "${{ steps.daskhub_chart.outputs.appVersion }}"
-          replace: "${{ steps.latest.outputs.tag }}"
+          # matches the dask and daskhub folders root files
+          include: "dask*/*"
+          find: '"${{ steps.dask_chart.outputs.appVersion }}"'
+          replace: '"${{ steps.latest.outputs.tag }}"'
```

Comment:
I made us search and replace the quotes as well to reduce the risk of miss-replacing something unrelated.